doc/thing/samples/basic-samples/tank.rst

```diff
@@ -0,0 +1,54 @@
+tank
+====
+
+Overview
+--------
+
+The tank example simulates a device showing the current tank volume.
+The purpose of this sample is to show how to register a floating-point
+value as a sensor using the KNoT Zephyr SDK.
+
```

Comment:
Please, make it clear that the value is automatically updated. And how it is updated

Reply:
Ok.